src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs

```diff
@@ -354,5 +355,26 @@ private void ApplyDaoTransition(Block block)
                 }
             }
         }
+
+        private void ApplyWithdrawals(Block block, IReleaseSpec spec)
```

Comment:
You shouldn't decode & apply withdrawals as long as the EIP is activated. Malicious peers can send you withdrawals before Shanghai hardfork

Reply:
We need to have tests that before withdarawals EIP we won't apply them